src/main/java/com/epam/repository/PersonDAO.java

```diff
@@ -3,6 +3,7 @@
 import com.epam.model.Person;
 import com.epam.model.PersonRoleEnum;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.jdbc.core.JdbcTemplate;
 import org.springframework.stereotype.Repository;
 import org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplate;
```

Comment:
Why we need this import?

Reply:
In this little pull request, we switched from NamedParameterJdbcTemplate to usual JdbcTemplate.